commcare_connect/templates/opportunity/opportunity_create.html

```diff
@@ -12,5 +12,28 @@
       {% crispy form %}
     </form>
 
+    <script>
+      window.onload = function () {
+        const applications = {{ applications|safe }};
+        const deliver_app_select = document.getElementById("deliver_app_select");
+
+        deliver_app_select.addEventListener("change", populateDeliverFormSelect);
+        const deliver_form_select = document.getElementById("deliver_form_select");
+        populateDeliverFormSelect();
+
+        function populateDeliverFormSelect() {
+          const selectedAppId =  deliver_app_select.options[deliver_app_select.selectedIndex].value;
+          const selectedApp = applications.filter(app => app.id === selectedAppId)[0];
+          deliver_form_select.innerHTML = "";
+
+          for (const form of selectedApp.forms) {
+            const form_option = document.createElement("option");
+            form_option.text = form.name;
+            form_option.value = form.id;
+            deliver_form_select.add(form_option);
+          }
+        }
+      }
+    </script>
```

Comment:
This code is fine - just wondering if you looked at using alpinejs? e.g. https://www.raymondcamden.com/2022/07/29/building-related-selects-in-alpinejs

Reply:
I took a look at alpineJS, I was already done with these changes by then.